R/package_skeleton.R

```diff
@@ -41,20 +41,23 @@ package_skeleton <- function(path = ".", git_init = TRUE, git_clone = NULL,
     git_init = FALSE
   }
   
-  if (description_template ) {
-    add_description(npath)
-  }
+
+  bcgovr_desc = list("Package" = sub('.*\\/', '', npath))
   
   ## Add in package setup files
-  devtools::setup(npath, rstudio = FALSE) 
+  devtools::setup(npath, rstudio = TRUE, description = bcgovr_desc) 
```

Comment:
I wonder if the more relevant question is: can you think of an instance where you are creating a package and wouldn't require .Rproj file? If you can then I will certainly make this change. If not though then maybe the rstudio and the description_template argument should both just be removed?